rest-api-model-ql/src/main/kotlin/org/modelix/sample/restapimodelql/LightModelClientWrapper.kt

```diff
@@ -0,0 +1,115 @@
+package org.modelix.sample.restapimodelql
+
+import University.Schedule.*
+import io.ktor.client.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.plugins.websocket.*
+import jetbrains.mps.lang.core.N_BaseConcept
+import kotlinx.coroutines.delay
+import org.modelix.client.light.LightModelClient
+import org.modelix.client.light.WebsocketConnection
+import org.modelix.metamodel.typed
+import org.modelix.model.repositoryconcepts.N_Module
+import org.modelix.model.repositoryconcepts.N_Repository
+import org.modelix.model.repositoryconcepts.models
+import org.modelix.model.repositoryconcepts.rootNodes
+import org.modelix.model.server.api.buildModelQuery
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
+
+val logger: Logger = LoggerFactory.getLogger("org.modelix.sample.restapimodelql.ModelServerLightWrapper")
+
+class LightModelClientWrapper(host: String = "localhost", port: Int = 48302, models: String) {
+
+    private var mpsModelName: String = models
```

Comment:
```suggestion
class LightModelClientWrapper(host: String = "localhost", port: Int = 48302, mpsModelName: String) {

    private var mpsModelName: String = mpsModelName
```

Reply:
dont know why it does not show but I already changed it to `modelNameSubstring`